site/app/templates/course/UploadCourseMaterialsForm.twig

```diff
@@ -145,10 +145,15 @@
                 if(document.getElementById('show_Some_Section_Selection').style.display == "none"){
                     sections = null;
                 }
-                var cmPath = document.getElementById('cm_path').innerHTML;
-                var requestedPath = document.getElementById("input-provide-full-path").value;
-                var sections_lock = document.getElementById('all_Sections_Showing_yes').checked === true;
-                makeSubmission(expandZip, hideFromStudents, cmPath, requestedPath, cmTime, sort, sections, sections_lock);
+                if(sections == null || sections.length > 0){
```

Comment:
It would be better to use `===` here instead of `==` to prevent JS from trying to cast the types automatically.